File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");   //ask the user for the height
    }
    while (!((height >= 1) && (height <= 8)));

    for (int i = 1; i <= height; i++)
    {
        for (int k = 1; k <= (height - i); k++)
        {
            printf(" ");    //prints the blank space of the triangle
        }
        for (int j = 1; j <= i; j++)
        {
            printf("#");    //prints the tyesriangle iterations
        }
        printf("\n");
    }
}

File: pset6/sentimental-mario-less/mario.py
from cs50 import get_int

# Ask the user for input
while True:
    number = get_int("Height: ")
    if (number >= 1 and number <= 8):
        break

# Start the
for i in range(1, number + 1):
    for j in range(number - i):
        print(" ", end="")
    for s in range(i):
        print("#", end="")
    print("")